cabot/metricsapp/models/elastic.py

```diff
@@ -193,20 +193,35 @@ def _parse_series(self, series, series_name=None):
         :param series_name: the name for the series ('key1.key2. ... .metric')
         :return: (series, (timestamp, datapoint)) pairs
         """
-        for subseries in series:
-            # If there are no more aggregations, we've reached the metric
+
+        islist = isinstance(series, list)
```

Comment:
changed `len(subseries) == 2` to `isinstance(subseries, tuple)` since the dictionary could be len 2